ui/pages/notifications-settings/notifications-settings.tsx

```diff
@@ -82,7 +81,7 @@ export default function NotificationsSettings() {
             ariaLabel="Back"
             iconName={IconName.ArrowLeft}
             size={ButtonIconSize.Sm}
-            onClick={() => history.push(NOTIFICATIONS_ROUTE)}
+            onClick={() => history.go(-1)}
```

Comment:
I don't see this used elsewhere in extension.
Can we use `history.back()`?

Reply:
it seems we should use `history.goBack()` 🤷

Reply:
Aha nice find!